extension/bootstrap.js

```diff
@@ -78,20 +74,18 @@ var PersonasPlusBootstrapAddon = {
         Cu.unload("resource://personas/modules/personas.js");
         this.requestRemovePrerequisites(data);
         if (reason == ADDON_DISABLE || reason == ADDON_UNINSTALL) {
-            // modules that come with Firefox
-            Cu.import("resource://gre/modules/XPCOMUtils.jsm");
-            // LightweightThemeManager may not be not available (Firefox < 3.6 or Thunderbird)
-            try {
-                Cu.import("resource://gre/modules/LightweightThemeManager.jsm");
-            } catch (e) {
-                LightweightThemeManager = null;
-            }
-            try {
-                LightweightThemeManager.forgetUsedTheme(LightweightThemeManager.currentTheme.id);
-            } catch (e) {
-                try {
-                    LightweightThemeManager.currentTheme = null;
-                } catch (e) {}
+            if(LightweightThemeManager.currentTheme){
+                var prompt = Components.classes["@mozilla.org/embedcomp/prompt-service;1"].getService(Components.interfaces.nsIPromptService);
+                if (prompt.confirm(null, "Personas Plus", "Would you like to remove the current theme?")) {
+                    try {
+                        Cu.import("resource://gre/modules/LightweightThemeManager.jsm");
+                        LightweightThemeManager.forgetUsedTheme(LightweightThemeManager.currentTheme.id);
```

Comment:
So, what happens if _Groovy Blue_ is the current theme?

If the answer is _yes_, either this line or line 245 will fail.
If the answer is _no_, _Groovy Blue_ will be removed because of line 245 despite the fact that the answer was _no_.